.github/workflows/integration-tests.yml

```diff
@@ -16,23 +16,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8.17", "3.11.4", "3.12.0b4"]
+        python-version: ["3.8.19", "3.11.9", "3.12.4", "3.13"]
```

Comment:
I think now we can skip the minor numbers

I.e. 3.8, 3.11, 3.12, 3.13

we do really care which patch release it would be.

With the previous action it wasn't possibility, and had to specify it to the level.

Reply:
I would prefer to keep everything binded.
It creates less confusion when refression happens.

Reply:
Well, on the other hand you might miss some, if it's pinned.

Such patch releases regression are quite rare, I don't recall seeing one in the last 5 years

Reply:
I agree that it's better to avoid specifying patch number - this is CI, if there is (unlikely) issue with new patch update, then CI should detect it.

Reply:
ok, removed patch numbers